openid-4-verifiable-presentations-1_0.md

```diff
@@ -811,14 +811,20 @@ Note: Some of the processing rules of the Presentation Definition and the Presen
 
 The Verifier has the choice of the following mechanisms to invoke a Wallet:
 
-- Custom URL scheme as an `authorization_endpoint` (for example, `openid4vp://` as defined in (#openid4vp-profile))
+- Custom URL scheme as an `authorization_endpoint` using `openid4vp://` as defined in (#openid4vp-custom-url-scheme)
 - Domain-bound Universal Links/App link as an `authorization_endpoint`
 - no specific `authorization_endpoint`, user scanning a QR code with Authorization Request using a manually opened Wallet, instead of an arbitrary camera application on a user-device (neither custom URL scheme nor Universal/App link is used)
 
 # Wallet Metadata (Authorization Server Metadata) {#as_metadata_parameters}
 
 This specification defines how the Verifier can determine Credential formats, proof types and algorithms supported by the Wallet to be used in a protocol exchange.
 
+## Authorization Endpoint Custom URL Scheme {#openid4vp-custom-url-scheme}
+
+The custom URL scheme representing the Authorization Endpoint defined in this specification is `openid4vp`.
+Implementations that use the custom URL scheme defined in this specification MUST configure their `authorization_endpoint` metadata parameter using the value `openid4vp://`, presenting the custom URL scheme along with the hier-part and separated by a colon, as defined in [@!RFC3986].
+
+
```

Comment:
I think there is a misunderstanding. openid4vp does not mandate supporting openid4vp. this specification only defines a static metadata config that can be used with openid4vp, but it is not mandatory to support it by any means.

```suggestion
## Authorization Endpoint and a Custom URL Scheme {#openid4vp-custom-url-scheme}

In this specification, the Authorization Endpoint can be a custom URL scheme.
```